xs2a-adapter-aspsp-registry/src/main/java/de/adorsys/xs2a/adapter/registry/LuceneAspspRepository.java

```diff
@@ -222,4 +223,24 @@ private Query getNameQuery(String name) {
         }
         return find(queryBuilder.build(), after, size);
     }
+
+    @Override
+    public byte[] getAllRecords() {
```

Comment:
It's not repository's responsibility to convert a collection of objects into a csv file